Check whether it exists 'input_file_name' parameter

djvicente opened the pull request:
Some changes were introduced in the impose_nodal_reference_temp process to check if it exists the parameter 'input_file_name'. This has been included given that in the Acombo Release#6 interface, this parameter was removed from the nodal reference process.

djvicente committed:
Check whether it exists 'input_file_name' parameter

Some changes were introduced in the impose_nodal_reference_temp process to check if it exists the parameter 'input_file_name'. This has been included given that in the Acombo Release#6 interface, this parameter was removed from the nodal reference process.

rubenzorrilla commented on applications/DamApplication/python_scripts/impose_nodal_reference_temperature_process.py:
```diff
@@ -15,7 +15,15 @@ def __init__(self, Model, settings ):
+
+        # This exception is introduced to check if the parameter "input_file_name" exixts. If not, it create it and define it as "".
+        # This could be changed when the Nodal Reference Temp (input file) process will be fixed. So far, in the ACOMBO Release 6, this parameter (input_file_name) was removed from the interface.
+        try:
```
I wouldn't use the `try` statement... You can use the `Has` method from `Parameters` object. Something like:
```py
if settings.Has("input_file_name"):
    input_file_name = settings["input_file_name"].GetString()
else:
    input_file_name = ""
```
Remember that `try` instruction tries to handle an error. Thus let's avoid it instead of tackling it ;).

djvicente replied:
Without a doubt, your proposal seems much better than mine. I did not know the Has method from Parameters object. I still have a lot to learn ;)
Thank you very much Ruben!

djvicente committed:
Change in the way to check if the parameter 'input_file_name' exists

This change has been proposed by Ruben Zorrilla. The method Has from the object Parameter fits perfectly to the purspose of this code.
